spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java

```diff
@@ -91,7 +97,12 @@ public TaskLauncher taskLauncher() {
 
 	@Bean
 	public PartitionHandler partitionHandler(TaskLauncher taskLauncher, JobExplorer jobExplorer) throws Exception {
-		MavenResource resource = MavenResource.parse("io.spring.cloud:partitioned-batch-job:1.0.0.BUILD-SNAPSHOT");
+
```

Comment:
if you use the `delegatingResourceLoader` here, you don't need to control the maven properties here (since they'd be applied to the `MavenResourceLoader` within the `delegatingResourceLoader` *and* you would get the benefit of being able to load from more than Maven, e.g. `file:` or `http:` URLs.